runtimes/pythonrt/worker_grpc_handler.go

```diff
@@ -157,82 +161,71 @@ func (s *workerGRPCHandler) Activity(ctx context.Context, req *pb.ActivityReques
 
 // ak functions
 
-func makeCallbackMessage(args []sdktypes.Value, kwargs map[string]sdktypes.Value) *callbackMessage {
+func rpcSyscall[Resp proto.Message](
+	rid string,
+	callf func(context.Context, sdkservices.RunSyscalls) (sdktypes.Value, error),
+	errf func(error) Resp,
+	vf func(sdktypes.Value) Resp,
+) (Resp, error) {
+	w.mu.Lock()
+	runner, ok := w.runnerIDsToRuntime[rid]
```

Comment:
If `w.runnerIdsToRuntime` is nil, this code can panic and then we don't unlock the mutex.